File: src/main/java/eu/mikroskeem/jutuskeem/commands/MainCommand.java
package eu.mikroskeem.jutuskeem.commands;

import com.google.inject.Inject;
import eu.mikroskeem.jutuskeem.Main;
import org.bukkit.command.Command;
import org.bukkit.command.CommandExecutor;
import org.bukkit.command.CommandSender;

public class MainCommand implements CommandExecutor {
    @Inject private Main plugin;

    @Override
    public boolean onCommand(CommandSender commandSender, Command command, String s, String[] strings) {
        if(command.getName().equals("jutuskeem")){
            plugin.reloadConfig();
            commandSender.sendMessage("Configuration reloaded");
        }
        return false;
    }
}

File: src/main/java/eu/mikroskeem/jutuskeem/commands/MainCommand.kt
package eu.mikroskeem.jutuskeem.commands

import eu.mikroskeem.jutuskeem.Main
import org.bukkit.command.Command
import org.bukkit.command.CommandExecutor
import org.bukkit.command.CommandSender

/**
 * @author Mark Vainomaa
 */
class MainCommand(private val plugin: Main) : CommandExecutor {
    override fun onCommand(sender: CommandSender, command: Command, label: String, args: Array<String>): Boolean {
        if(command.name == "jutuskeem") {
            plugin.reloadConfig()
            plugin.messages.reloadMessages()
            plugin.messages.sendMessage(sender, "configuration-reloaded", null)
        }
        return false
    }
}